src/components/anchor-navigation/anchor-navigation.test.tsx

```diff
@@ -0,0 +1,364 @@
+import React from "react";
+import { render, screen, act } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import Textbox from "../textbox";
+import {
+  AnchorNavigation,
+  AnchorNavigationItem,
+  AnchorSectionDivider,
+} from ".";
+import { AnchorNavigationProps } from "./anchor-navigation.component";
+
+const SECTION_VISIBILITY_OFFSET = 200;
```

Comment:
**suggestion**: as this is only used once, we can probably just use the numerical value on line `248` reducing the need for a const being defined here

Reply:
not so sure I agree with this one - "magic numbers" are considered a bad thing, the const is better even if it's only used once, as it's easier for readers of this code to see what's going on (rather than seeing a hard-coded 200 in a test and wondering what it means), and also easier for future maintenance to know which value to update if UI changes ever mean this has to be updated, rather than hunting for a particular number in a particular test which needs updating.

Happy to hear counter-arguments if I've missed something though!

Reply:
I suppose the const definition could be moved to be just inside that one test, since it's only used once - I'd be perfectly happy with that.